src/Tizen.Multimedia/Common/Display.cs

```diff
@@ -137,13 +137,32 @@ public Display(Window window)
         /// </remarks>
         /// <since_tizen> 3 </since_tizen>
         public Display(NUI.Window window)
+            : this (window, false)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Display"/> class with a <see cref="NUI.Window"/> class.
+        /// </summary>
+        /// <param name="window">A <see cref="NUI.Window"/> to display.</param>
+        /// <param name="uiSync">A value indicates whether video and UI is in sync or not.</param>
+        /// <remarks>
+        /// The <see cref="NUI.Window.BackgroundColor"/> must be <see cref="NUI.Color.Transparent"/>
+        /// for the <see cref="Display"/> to be rendered correctly.<br/>
+        /// UI sync is only for <see cref="Tizen.Multimedia.Player"/> and
+        /// <see cref="Tizen.Multimedia.Player.DisplaySettings"/> will not work in UI sync mode.
```

Comment:
For simple \<see> to work the name should be visible to the compiler, you can add T: to fix it, or use the following snippet to have the full name of DisplaySettings property
```suggestion
        /// UI sync is only for <see cref="T:Tizen.Multimedia.Player"/> and
        /// [](xref:Tizen.Multimedia.Player.DisplaySettings?displayProperty=fullNam
e) will not work in UI sync mode.
```